### ISSUE TITLE
How to build this?

### ISSUE THREAD

**Issue by mihaifm** [macote/Winook#35]
Hi, any chance for some build instructions?

I tried compiling the sln, but getting this error when running the test app.

![image](https://user-images.githubusercontent.com/981184/180822059-df7cdc38-4752-415f-826b-896436c0f6bf.png)

Do I need to build the asio lib or do anything else?

**Comment by macote** [comment]
Please provide the following: Windows version, VS version and which test app you were trying.

You don't need to build the support files (C++), they're included in folder `src\Winook\winook.support`.

**Comment by mihaifm** [comment]
Hi, sorry for the late reply, I am using Windows 11, VS 2019 and was trying to run the Winook.Desktop.Test app.

thanks

**Comment by macote** [comment]
Hi Mihai,

I found the problem: in Windows 11, Notepad was replaced by a new application available in the Microsoft Store. It seems that the executables located where the Winook test app points to are stubs that launch the new version of Notepad.

On my machine, it works when I wait a bit for the application to start (and show the main window) before executing `InstallAsync()`. I use a breakpoint and wait 1 second.
